### PR TITLE
AP-3531 better differentiate substantive from emergency on default LOS pages

### PR DESCRIPTION
**Pull request by RoseSAK**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-3531)

Updated the pages for substantive LOS default and emergency LOS default to make it clearer that they are separate and different pages
This also fixes the issue with the error message not appearing in the right place on the page (it was appearing directly below the header instead of above the radio buttons)

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by cpjmcquillan — APPROVED** [review]

**Review by colinbruce — APPROVED** [review]

File: app/views/providers/proceeding_loop/emergency_defaults/show.html.erb [hunk 1]
```diff
@@ -6,19 +6,22 @@
     ) do |form| %>
   <%= page_template template: :basic, form: form do %>
     <span class="govuk-caption-l"><%= position_in_array(@proceeding) %></span>
-    <h1 class="govuk-heading-xl"><%= @proceeding.meaning %></h1>
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-4"><%= @proceeding.meaning %></h1>
+
+    <h2 class="govuk-heading-m"><%= t('.h2_header') %></h2>
+
+    <div class="govuk-!-padding-1"></div>
```
Comment (naseberry) [inline app/views/providers/proceeding_loop/emergency_defaults/show.html.erb:13]: Would be much cleaner to add the spacing class name `govuk-!-padding-1` to `h2` and remove this empty element, that only purpose is to apply spacing?

File: app/views/providers/proceeding_loop/emergency_defaults/show.html.erb [hunk 2]
```diff
@@ -6,19 +6,22 @@
...
+
+    <p><strong><%= t('.los_header') %></strong> <%= form.object.emergency_level_of_service_name %></p>
+    <p><strong><%= t('.scope_header') %></strong> <%= form.object.delegated_functions_scope_limitation_meaning %></p>
+    <p><strong><%= t('.scope_description_header') %></strong> <%= form.object.delegated_functions_scope_limitation_description %></p>
+
+    <div class="govuk-!-padding-2"></div>
```
Comment (naseberry) [inline app/views/providers/proceeding_loop/emergency_defaults/show.html.erb:19]: ditto.
Would an alternate approach to empty element for spacing be better?

File: app/views/providers/proceeding_loop/substantive_defaults/show.html.erb [hunk 3]
```diff
@@ -6,19 +6,22 @@
     ) do |form| %>
   <%= page_template template: :basic, form: form do %>
     <span class="govuk-caption-l"><%= position_in_array(@proceeding) %></span>
-    <h1 class="govuk-heading-xl"><%= @proceeding.meaning %></h1>
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-4"><%= @proceeding.meaning %></h1>
+
+    <h2 class="govuk-heading-m"><%= t('.h2_header') %></h2>
+
+    <div class="govuk-!-padding-1"></div>
```
Comment (naseberry) [inline app/views/providers/proceeding_loop/substantive_defaults/show.html.erb:13]: ditto

File: app/views/providers/proceeding_loop/substantive_defaults/show.html.erb [hunk 4]
```diff
@@ -6,19 +6,22 @@
...
+
+    <p><strong><%= t('.los_header') %></strong> <%= form.object.substantive_level_of_service_name %></p>
+    <p><strong><%= t('.scope_header') %></strong> <%= form.object.substantive_scope_limitation_meaning %></p>
+    <p><strong><%= t('.scope_description_header') %></strong> <%= form.object.substantive_scope_limitation_description %></p>
+
+    <div class="govuk-!-padding-2"></div>
```
Comment (naseberry) [inline app/views/providers/proceeding_loop/substantive_defaults/show.html.erb:19]: ditto

**Pull request merged**